README.md

```diff
@@ -468,6 +469,170 @@ ANSWERS.addComponent('DirectAnswer', {
+    value: "+11234567890",
+    fieldType: "phone" 
+  },
+  relatedItem: { 
+    verticalConfigId: 'people',
+    data: { 
+      id: "Employee-2116",
+      type: "ce_person",
+      fieldValues: {
+        description: "This is the description field.",
+        name: "First Last",
+        firstName: "First",
+        lastName: "Last",
+        mainPhone: "+1234567890",
+      }
+    }
+  }
+}
+```
+
+A Direct Answer custom card needs a corresponding template. This can be added either inline by changing the component's constructor to:
+
+```js
+    constructor(config, systemConfig) {
+      super(config, systemConfig);
+      this.setTemplate(`<div> your template here </div>`)
+    }
+```
+
+Or by including a custom template bundle, and adding:
+
+```js
+  static defaultTemplateName () {
+    return 'YourTemplateName';
+  }
+```
+
+Where 'YourTemplateName' is the name of the template registered onto the Handlebars renderer.
+
+The example custom Direct Answer card will use the Handlebars template below.
+
+```hbs
+  <div class="customDirectAnswer">
+    <div class="customDirectAnswer-type">
+      {{type}}
+    </div>
+    <div class="customDirectAnswer-value">
+      {{#each customValue}}
+        {{#if url}}
+          {{> valueLink }}
+        {{else}}
+          {{{this}}}
+        {{/if}}
+      {{/each}}
+    </div>
+  </div>
+
+  {{#*inline 'valueLink'}}
+  <a class="customDirectAnswer-fieldValueLink" href="{{{url}}}"
+    {{#if @root/eventType}}data-eventtype="{{@root/eventType}}"{{/if}}
+    {{#if @root/eventOptions}}data-eventoptions='{{{ json @root/eventOptions }}}'{{/if}}>
+    {{{displayText}}}
+  </a>
+  {{/inline}}
+```
+
+This is an example of a possible custom Direct Answer card. It has custom rendering that depends on
+the fieldType of the directAnswer, as well as a a custom analytics event.
+
+```js
+  class CustomDirectAnswerClassName extends ANSWERS.Component {
+    constructor(config, systemConfig) {
+      // If you need to override the constructor, make sure to call super(config, systemConfig) first.
+      super(config, systemConfig);
+    }
+
+    /**
+     * The setState method allows you to pass variables directly into your template.
+     * Here, data is the directAnswer data from the query.
+     * Below, we pass through a custom direct answers value, customValue.
+     * @param {Object} data
+     * @returns {Object}
+     */ 
+    setState(data) {
+      const { type, answer, relatedItem } = data;
+      this.associatedEntityId = data.relatedItem && data.relatedItem.data && data.relatedItem.data.id;
+      this.verticalConfigId = data.relatedItem && data.relatedItem.verticalConfigId;
+      return super.setState({
+        ...data,
+        customValue: this.getCustomValue(answer),
+        eventType: 'CUSTOM_EVENT',
+        eventOptions: {
+          searcher: "UNIVERSAL",
+          verticalConfigId: this.verticalConfigId,
+          entityId: this.associatedEntityId,
+        }
+      });
+    }
+
```

Comment:
For the Direct Answer card feedback, they'd need to set up the click listeners in an onMount. Do you think it's worth adding one here in this example? This is what the theme onMount looks like: https://github.com/yext/answers-hitchhiker-theme/blob/master/directanswercards/card_component.js#L40

Reply:
sure added a basic onMount. I tried doing the same form thing that the theme does, but then after looking at it again it just seemed way too complicated for this case.

Reply:
Amazing lgtm